moc_openstack_api_reporting.py

```diff
@@ -734,7 +734,7 @@ def main():
 
     vol_keys = list(cluster_state["vol_timestamps"].keys())
     for key in vol_keys:
-        if cluster_state["vol_timestamps"][key]["updated"] == 0:
+        if "updated" in list(cluster_state["vol_timestamps"][key].keys()) and cluster_state["vol_timestamps"][key]["updated"] == 0:
```

Comment:
You could use the get method on the dictionary.

`cluster_state["vol_timestamps"][key].get("updated") == 0`

because `cluster_state["vol_timestamps"][key].get("updated")` would return None when "updated" doesn't exist and the comparison would then correctlya fail.